src/object/implementation/program_storage_impl.cpp

```diff
@@ -131,7 +131,7 @@ Program* CProgramStorageObjectImpl::CloneProgram(Program* program)
 
     // TODO: Is there any reason CScript doesn't have a function to get the program code directly?
     auto edit = MakeUnique<Ui::CEdit>();
-    edit->SetMaxChar(Ui::EDITSTUDIOMAX);
+    edit->SetInfinite();
```

Comment:
Not really necessary as infinite is now the default
(also applies to all other occurrences of this but I won't copy-paste this comment a billion times)